README.rst

```diff
@@ -42,39 +42,15 @@ Conformance
 
 This library aims to conform with the following:
 
-- `JSON-LD 1.0`_,
-  W3C Recommendation,
-  2014-01-16, and any `errata`_
-- `JSON-LD 1.0 Processing Algorithms and API <JSON-LD 1.0 API_>`_,
-  W3C Recommendation,
-  2014-01-16, and any `errata`_
-- `JSON-LD 1.0 Framing <JSON-LD 1.0 Framing_>`_,
-  Unofficial Draft,
-  2012-08-30
```

Comment:
The 1.1 support update should still work for most 1.0 data right?  There are some exceptions but I didn't think enough to remove this text.  I imagine the old framing and 1.1 CG draft support is more questionable at this point.

Reply:
The 1.1 specs are compatible with the 1.0 steps; it's redundant to list all previous versions of specifications supported. This includes Framing 1.1 being compatible with the unofficial 1.0 draft.